src/firmware-update.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: base
+description: Zigbee and Thread USB stick by the creators of Home Assistant
+---
+
+<div class="container">
+  <div>
+    <h1>Firmware update</h1>
+    <p>First things first: a firmware upgrade is not required for SkyConnect at this point.</p>
+    <p>However: We're working on making firmware updates available via Home Assistant.
```

Comment:
IMO, that is not a "However", that is just a different fact.

```suggestion
    <p>We're also working on making firmware updates available via Home Assistant. So you will receive updates automatically in the future.
```